.travis.yml

```diff
@@ -71,24 +69,25 @@ matrix:
 
     - os: osx
       osx_image: xcode9.2
-      env: PROJECT_DIR=examples/acf TOOLCHAIN=osx-10-13-cxx14
+      env: PROJECT_DIR=examples/acf TOOLCHAIN=osx-10-13-cxx14 VERBOSE=0
 
-    # WebP i386 build is broken
+    # Upload from local machine
+    # FIXME: modules/core/src/parallel_impl.cpp:396:21: error: implicitly declaring library function '_mm_pause' with type 'void ()'
     # - os: osx
     #   osx_image: xcode9.2
-    #   env: PROJECT_DIR=examples/acf TOOLCHAIN=ios-nocodesign-11-2-dep-9-3
+    #   env: PROJECT_DIR=examples/acf TOOLCHAIN=ios-nocodesign-11-2-dep-9-3 VERBOSE=0
 
     # }
 
-    # Extra {
+    # Extra { 
 
     - os: osx
       osx_image: xcode9.2
-      env: PROJECT_DIR=examples/acf TOOLCHAIN=ios-nocodesign-11-2-dep-9-3-armv7
+      env: PROJECT_DIR=examples/OpenCV TOOLCHAIN=ios-nocodesign-11-2-dep-9-3-armv7 VERBOSE=0
```

Comment:
`s,OpenCV,acf,`

Reply:
so effectively there is no real changes introduced by this patch (?)

Reply:
I still see no real changes here like adding new toolchain.

Reply:
I was missing your point.  I see now, a recent `pkg.template` was merged in yesterday here : https://github.com/ingenue/hunter/commit/bb365a67cf01b0f8e4a7dec84e14e4fda780a1bf#diff-354f30a63fb0907d4ad57269548329e3

I applied the same changes for `test.acf` tests and sent them back here without realizing they were already applied.